### ISSUE TITLE
Event History

### ISSUE THREAD

**Issue by abuneri** [Injabie3/LLSIF-Helper#13]
I feel like this would be good practice for us, as well as useful for the app in general. But we should think of a way to store all the data for each event, could be used as a progress tracker, or just for users to be able to see how they can be more efficient in future events.

This is where we would be able to take some Data Structures (eg. Linked List) and use them!

**Comment by Injabie3** [comment]
I agree :+1: . In the Excel spreadsheet I have going on, I'm keeping track of stuff for score match, so I think I can help implement that for score match :)

**Comment by abuneri** [comment]
nice, i have a few things in mind that we could keep and present as history for all the events, these are more high-level, general stats. but we can think of what exactly we want soon enough!
